app/src/main/java/com/example/tungcon/demo/movie/IamgeMovie/ItemMovieActivity.java

```diff
@@ -0,0 +1,52 @@
+package com.example.tungcon.demo.movie.IamgeMovie;
+
+import android.content.Context;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.bumptech.glide.Glide;
+import com.example.tungcon.demo.R;
+import com.example.tungcon.demo.data.model.Movie;
+
+public class ItemMovieActivity extends AppCompatActivity {
+    private static final String EXTRA_MOVIE = "EXTRA_MOVIE";
+    private ImageView mImageMovie;
+    private TextView mTextViewMovie;
+    private Movie mMovie;
+
+    public static Intent getDataIntent(Context context, Movie movie) {
+        Intent intent = new Intent(context, ItemMovieActivity.class);
+        intent.putExtra(EXTRA_MOVIE, movie);
+        return intent;
+    }
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_item_movie);
+        intViews();
+        getMovieFromIntent();
+    }
+
+    private void getMovieFromIntent() {
+        Intent intent = getIntent();
+        mMovie = (Movie) intent.getSerializableExtra(EXTRA_MOVIE);
+        showMovie();
+    }
+
+    private void showMovie() {
```

Comment:
method này là showMovie thì input nên cần có 1 movie 
```suggestion
    private void showMovie(Movie movie) {
```